test/bad_dtls_test.c

```diff
@@ -548,8 +548,8 @@ static int test_bad_dtls(void)
 
         if (!TEST_true(send_record(rbio, SSL3_RT_APPLICATION_DATA, tests[i].seq,
                                    &tests[i].seq, sizeof(uint64_t)))) {
-            TEST_error("Failed to send data seq #0x%lx (%d)\n",
-                       tests[i].seq, i);
+            TEST_error("Failed to send data seq #0x%x%08x (%d)\n",
+                       (int)(tests[i].seq >> 32), (int)(tests[i].seq & 0x0FFFFFFFF), i);
```

Comment:
Thing is that standard doesn't specify what happens when you cast from unsigned to signed [if value can't be presented], only from signed to unsigned.

In case you sense contradiction with suggestion to cast agesec. Well, it does need overhaul (as already mentioned in #3105). But it's actually implicitly ensured that it's small enough to be safe for cast.